app/controllers/movies_controller.rb

```diff
@@ -1,5 +1,5 @@
 class MoviesController < ApplicationController
   def index
-    @movies = Movie.order(year: :desc, priority: :asc, title: :asc).page(params[:page])
+    @movies = Movie.order(as_assistant: :asc, year: :desc, priority: :asc, title: :asc).page(params[:page])
```

Comment:
Does this work? All film where `as_assistant` equal `true` are push at the end?

Reply:
Yes, because as_assistant: :asc is the first parameter of . order().

Reply:
OK I wasn't sure that `false` values would be at the beginning 👍